src/js/media-airplay-button.ts

```diff
@@ -36,6 +36,10 @@ slotTemplate.innerHTML = /*html*/ `
     }])) slot[name=tooltip-exit] {
       display: none;
     }
+
+    :host([${MediaUIAttributes.MEDIA_AIRPLAY_UNAVAILABLE}]) {
+      display: none;
+    }
```

Comment:
this has to be reverted. the users themselves have to set that CSS in there page.